compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1266,52 +1266,29 @@ LogicalResult WinogradOutputTransformOp::reifyResultShapes(
 //===----------------------------------------------------------------------===//
 
 LogicalResult AttentionOp::verify() {
-  Operation *op = getOperation();
+  AttentionOp attnOp = *this;
```

Comment:
[No action item required] Technical, we don't need to capture `*this`. We can just do `emitOpError()` instead of (attnOp->emitOpError). I forgot why we have this style in the file. I'm just putting a note that it is not needed.